src/game/common/const.mjs

```diff
@@ -94,6 +94,7 @@ export const GameTreasure = {
   tio: { type: 'tio', label: 'ar.io (tIO)', tile: 2, value: 100, baseVal: 150_000_000, denomination: 6 },
   war: { type: 'war', label: 'Wrapped AR', tile: 3, value: 300, baseVal: 6_000_000_000, denomination: 12 },
   trunk: { type: 'trunk', label: 'TRUNK', tile: 4, value: 100, baseVal: 66, denomination: 3 },
+  rsg: { type: 'rsg', label: 'RSG', tile: 5, value: 100, baseVal: 1, denomination: 0 },
```

Comment:
when someone collects war he gets 300 cboinc
when tio then 100
basically value says how much cbcoins you get
in case of rsg this should be... 0?